dev.zed.Zed.metainfo.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>dev.zed.Zed</id>
+  <name>Zed</name>
+  <summary>High-performance, multiplayer code editor from the creators of Atom and Tree-sitter.</summary>
+  <description>
```

Comment:
Missing developer tag https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/#developer-name